src/client.rs

```diff
@@ -18,6 +18,24 @@ pub trait Connectable {
     fn get_urls(self) -> Vec<String>;
 }
 
+impl Connectable for (&str, u16) {
+    fn get_urls(self) -> Vec<String> {
+        return vec![format!("{}:{}", self.0, self.1)];
+    }
+}
+
+impl Connectable for &[(&str, u16)] {
```

Comment:
These two should allow consumers to either repeatedly call `add_server` with a single tuple, or with a slice of tuples. That'll take care of host:port specification.